components/emoji_picker/emoji_picker.jsx

```diff
@@ -554,43 +554,8 @@ export default class EmojiPicker extends React.PureComponent {
             return this.sortEmojis(emojis);
         }
 
-        const emojis = this.state.categories[category.name].emojiIds.map((emojiId) =>
-            this.convertRecentEmojiToUserSkinTone(category, emojiId));
-
-        return this.filterDistinctEmojis(emojis);
-    }
-
-    getEmojiByUnicode = (unicode) => {
-        const emojiIndex = Emoji.EmojiIndicesByUnicode.get(unicode.toLowerCase());
-        return Emoji.Emojis[emojiIndex];
-    }
-
-    convertRecentEmojiToUserSkinTone = (category, emojiId) => {
```

Comment:
Is this reverting the code that makes recent emojis always fit the user's selected skintone, or does that still work?

Reply:
i improved the old code for recent ones to use skintone

Reply:
Yes i improve the code to selected skin tone only in recent section of emoji picker

Reply:
also due to git diff, code seems to be all the place, please check out the file in single mode if its gets too confusing with code block end etc

Reply:
please try to check out the file in the single mode, since git diff shows too much mess around